homeassistant/components/tessie/services.yaml

```
@@ -0,0 +1,24 @@
+set_navigation:
+  target:
+    entity:
+      - integration: tessie
+        domain: device_tracker
+  fields:
+    destination:
+      example: "1600 Amphitheatre Pkwy, Mountain View, CA"
+      selector:
+        text:
+    latitude:
+      selector:
+        number:
+          min: -90
+          max: 90
+    longitude:
+      selector:
+        number:
+          min: -180
+          max: 180
```

This would probably be better as a `location` selector.

```suggestion
    gps:
      selector:
        location:
          radius: false
```

---

See for reference: https://github.com/home-assistant/core/blob/dev/homeassistant/components/teslemetry/services.py#L105-L133

---

Indeed! I have now corrected this. Thank you.